src/main/scala/mesosphere/marathon/core/launchqueue/LaunchQueueModule.scala

```diff
@@ -0,0 +1,49 @@
+package mesosphere.marathon.core.launchqueue
+
+import akka.actor.{ ActorRef, Props }
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.launchqueue.impl.{
+  ActorLaunchQueue,
+  AppTaskLauncherActor,
+  LaunchQueueActor,
+  RateLimiter,
+  RateLimiterActor
+}
+import mesosphere.marathon.core.leadership.LeadershipModule
+import mesosphere.marathon.core.matcher.manager.OfferMatcherManager
+import mesosphere.marathon.core.task.bus.TaskStatusObservables
+import mesosphere.marathon.state.{ AppDefinition, AppRepository }
+import mesosphere.marathon.tasks.{ TaskFactory, TaskTracker }
+
+class LaunchQueueModule(
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.